libs/blocks/accordion/accordion.js

```diff
@@ -116,12 +156,57 @@ function populateMedia(accordion, id, num, collection) {
+  const closeIcon = '<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill="currentcolor" fill-rule="evenodd" clip-rule="evenodd" d="M13.6113 12L17.2647 8.34668C17.71 7.90186 17.71 7.18018 17.2647 6.73535C16.8193 6.29004 16.0986 6.29004 15.6533 6.73535L12 10.3887L8.34668 6.73535C7.90137 6.29004 7.18066 6.29004 6.73535 6.73535C6.29004 7.18017 6.29004 7.90185 6.73535 8.34668L10.3887 12L6.73535 15.6533C6.29004 16.0981 6.29004 16.8198 6.73535 17.2647C6.95801 17.4873 7.24951 17.5986 7.54101 17.5986C7.83251 17.5986 8.12402 17.4873 8.34667 17.2647L12 13.6113L15.6533 17.2647C15.876 17.4873 16.167 17.5986 16.459 17.5986C16.751 17.5986 17.042 17.4873 17.2646 17.2647C17.7099 16.8198 17.7099 16.0982 17.2646 15.6533L13.6113 12Z"/></svg>';
+  const config = getConfig();
+  const expandText = await replaceKey('expand-all', config);
+  const collapseText = await replaceKey('collapse-all', config);
+  const expandBtn = createTag('button', { class: 'expand-btn con-button button-l', 'aria-pressed': 'false' }, expandText);
+  expandBtn.insertAdjacentHTML('beforeend', chevronIcon);
+  const collapseBtn = createTag('button', { class: 'collapse-btn con-button button-l', 'aria-pressed': 'false' }, collapseText);
+  collapseBtn.insertAdjacentHTML('beforeend', closeIcon);
+  const container = createTag('div', { class: 'accordion-expand-all foreground' }, [expandBtn, collapseBtn]);
+  const toggleAll = (targetBtn, action) => {
+    if (targetBtn.getAttribute('aria-pressed') === 'true') return;
+    targetBtn.setAttribute('aria-pressed', 'true');
+    targetBtn.classList.remove('fill');
+    targetBtn.classList.add('disabled');
+    const siblingBtn = targetBtn.nextElementSibling || targetBtn.previousElementSibling;
+    siblingBtn.setAttribute('aria-pressed', 'false');
+    siblingBtn.classList.remove('disabled');
+    siblingBtn.classList.add('fill');
+    if (action === 'expand') {
+      accordionItems.forEach(({ dt, dd }) => openPanel(dt.querySelector('button'), dd));
+      if (!isEditorial) return;
+      [...mediaEl.childNodes].forEach((node, idx) => {
+        if (idx === 0) {
+          node.classList.add('expanded');
+          const video = node.querySelector('video');
+          if (video) playVideo(node.querySelector('video'));
+          return;
+        }
+        node.classList.remove('expanded');
+      });
+      return;
+    }
+    accordionItems.forEach(({ dt, dd }) => closePanel(dt.querySelector('button'), dd));
+    if (!isEditorial) return;
+    const video = mediaEl.querySelector('.expanded video');
+    if (video) pauseVideo(video);
+  };
```

Comment:
Possible to move into its own function for clarity and composability?

Reply:
This would add several extra params, since I'm referencing variables from the parent function.